docs/devices_api.yml

```diff
@@ -180,9 +171,27 @@ definitions:
         description: Request ID (same as in X-MEN-RequestID header).
         type: string
     example:
-      application/json:
-          error: "failed to decode device group data: JSON payload is empty"
-          request_id: "f7881e82-0492-49fb-b459-795654e7188a"
+      error: "failed to decode device group data: JSON payload is empty"
+      request_id: "f7881e82-0492-49fb-b459-795654e7188a"
+  DeploymentStatus:
+    type: object
+    properties:
+      status:
+        type: string
+        enum:
+          - installing
+          - downloading
+          - rebooting
+          - success
+          - failure
+          - already-installed
+      substate:
```

Comment:
Maybe we miss noartifact here?

Reply:
Well spotted!
I was about to say yes, but inspecting the code this status is a special case which is reported exclusively by the backend.

Reply:
thanks for checking!